code/game/objects/items/tanks/tanks.dm

```diff
@@ -23,7 +23,6 @@
 	throw_speed = 1
 	throw_range = 4
 	custom_materials = list(/datum/material/iron = 500)
-	actions_types = list(/datum/action/item_action/set_internals)
```

Comment:
uh, internals seem like something that should probably still have an action button, because new players suffocating because they can't figure out how to turn on their internals in time is a Bad Thing (TM)

Reply:
There's a button for that and exosuit thermoregulators when wearing one that is right below your health doll.
The one for internals is always present. You can just left click on that, I don't see why we should have two on screen at any time.

Reply:
fair enough, then

we might have an increase in the number of accidental carbon dioxide self-suffocations, though

Reply:
Anyone dumb enough to suffocate themselves with a co2 tank or jetpack is going to suffocate themselves with that source unless I literally make it impossible to do so. 
Some people deserve to die when they do something dumb, and that's their fault.
Don't stick a fork in an APC and come crying to me when you get shocked.

Reply:
Personally I would argue for removing the internals button and keeping the action button, but def kill one of them